ads/search.py

```diff
@@ -381,10 +384,18 @@ def __init__(self, query_dict=None, q=None, fq=None, fl=DEFAULT_FIELDS,
                 # cursors require unique field in the sort
                 if "id" not in sort and start is None:
                     sort = "{},id desc".format(sort)
+            if hl is not None:
+                hl_fl = [i for i in set(hl) if i in ['abstract', 'title', 'body']]
```

Comment:
Consider throwing a warning or exception if a non-supported value for `hl` is given -- I don't feel strongly one way or another personally.